src/utils/styles/globalStyles.css

```diff
@@ -1,3 +1,24 @@
 a {
   color: #9d4ba3;
 }
+.my-node-enter {
```

Comment:
Move these out of global styles, which are prone to breakage in the future. Instead, you can tell CSSTransition which classes to apply -- see [docs](http://reactcommunity.org/react-transition-group/css-transition#CSSTransition-prop-classNames). That should enable our typical use of `makeStyles`.

In addition, rename "my-node" to something more meaningful.